domain/src/main/java/org/oppia/android/domain/oppialogger/loguploader/LogUploadWorkManagerInitializer.kt

```diff
@@ -118,6 +131,10 @@ class LogUploadWorkManagerInitializer @Inject constructor(
     val workManager = WorkManager.getInstance(context)
     logUploader.enqueueWorkRequestForEvents(workManager, workRequestForUploadingEvents)
     logUploader.enqueueWorkRequestForExceptions(workManager, workRequestForUploadingExceptions)
+    logUploader.enqueueWorkRequestForPerformanceMetrics(
```

Comment:
Is there a test to verify this is working correctly?

Reply:
added.